src/main/java/com/hotjava/app/hotjava/dao/VehicleDAOStub.java

```diff
@@ -10,7 +10,7 @@
 @Profile("test")
 public class VehicleDAOStub implements IVehicleDAO {
 
-    Map<Integer, Vehicle> allVehicles = new HashMap<>();
+    Map<Long, Vehicle> allVehicles = new HashMap<>(); // changed to long.
```

Comment:
1) It's highly unlikely that there will be more than 2.1 billion vehicles, so Integer is a preferrable data type here.
2) Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.